onboarding/github.md

```diff
@@ -0,0 +1,23 @@
+# GitHub
+
+#### Organisation
+
+NeuroPoly has a github organisation:
+https://github.com/neuropoly
+
+#### Purpose
+
+
+The purpose of this organization is to host repos that are related to NeuroPoly's projects. Eg: if you publish a paper, you could put the code+data there and link to it in your paper.
+
+#### Philosophy
+
+The philosophy of NeuroPoly's lab is to do open science, which means that repos should be public. There are some exception: confidential data/credentials, highly-top-secret scientific research, etc.
```

Comment:
It's not clear to me what our philosophy of github is. Sometimes we make new orgs, sometimes we don't, sometimes we pay for them and sometimes we don't. Maybe it makes more sense to document the purpose of each org using a [org README](https://github.blog/changelog/2021-09-14-readmes-for-organization-profiles/) that will show on the front page of the org, and leave it.

At least, we definitely can't host data on GitHub. It's too expensive. We don't even do it for https://github.com/spine-generic/spine-generic/; that stuff is actually hosted on Amazon, which we're paying for; the GitHub repo is just an index into the Amazon servers.